{"uid":"7b87e82210905d0f","name":"test_01_sale_in_shop_product","fullName":"tests.test_1_sale_in_shop.test_1_sale_in_shop#test_01_sale_in_shop_product","historyId":"61c570a5c80e9195a1325ae1db2218fc","time":{"start":1747748723645,"stop":1747748723645,"duration":0},"status":"failed","statusMessage":"AssertionError: Текущий URL не соответствует ожидаемому. Ожидалось: 'https://erp-test.karman24.ru/', Фактически: 'https://erp-test.karman24.ru/auth'","statusTrace":"login_page = <pages.login_page.admin_login.AdminLogin object at 0x000001C66F3F9190>\n\n    @pytest.fixture(scope=\"function\")\n    def open_crm_as_admin(login_page):\n        login_page.open_page()\n        with allure.step('enter correct login and password'):\n            login_page.fill_login_form_good('krivko.su@codeagency.ru', 'DLNKsfd3214$%23')\n        with allure.step('Check correct url'):\n>           login_page.check_expected_url('https://erp-test.karman24.ru/')\n\nconftest.py:92: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.login_page.admin_login.AdminLogin object at 0x000001C66F3F9190>\nexpected_url = 'https://erp-test.karman24.ru/'\n\n    def check_expected_url(self, expected_url):\n        \"\"\"\n        Проверяет, что текущий URL соответствует ожидаемому.\n        :param expected_url: Ожидаемый URL.\n        \"\"\"\n        current_url = self.driver.current_url\n>       assert current_url == expected_url, \\\n            f\"Текущий URL не соответствует ожидаемому. Ожидалось: '{expected_url}', Фактически: '{current_url}'\"\nE       AssertionError: Текущий URL не соответствует ожидаемому. Ожидалось: 'https://erp-test.karman24.ru/', Фактически: 'https://erp-test.karman24.ru/auth'\n\npages\\base_page.py:184: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":16,"retriesStatusChange":true,"beforeStages":[{"name":"driver","time":{"start":1747748723648,"stop":1747748727367,"duration":3719},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"login_page","time":{"start":1747748727367,"stop":1747748727442,"duration":75},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"open_crm_as_admin","time":{"start":1747748727442,"stop":1747748768257,"duration":40815},"status":"failed","statusMessage":"AssertionError: Текущий URL не соответствует ожидаемому. Ожидалось: 'https://erp-test.karman24.ru/', Фактически: 'https://erp-test.karman24.ru/auth'\n","statusTrace":"  File \"C:\\Users\\Stass\\PycharmProjects\\PycharmProjects\\tests_ui_karman\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Stass\\PycharmProjects\\PycharmProjects\\tests_ui_karman\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Stass\\PycharmProjects\\PycharmProjects\\tests_ui_karman\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Stass\\PycharmProjects\\PycharmProjects\\tests_ui_karman\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Stass\\PycharmProjects\\PycharmProjects\\tests_ui_karman\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Stass\\karman_test\\conftest.py\", line 92, in open_crm_as_admin\n    login_page.check_expected_url('https://erp-test.karman24.ru/')\n  File \"C:\\Users\\Stass\\karman_test\\pages\\base_page.py\", line 184, in check_expected_url\n    assert current_url == expected_url, \\\n","steps":[{"name":"enter correct login and password","time":{"start":1747748736197,"stop":1747748768248,"duration":32051},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"Check correct url","time":{"start":1747748768249,"stop":1747748768255,"duration":6},"status":"failed","statusMessage":"AssertionError: Текущий URL не соответствует ожидаемому. Ожидалось: 'https://erp-test.karman24.ru/', Фактически: 'https://erp-test.karman24.ru/auth'\n","statusTrace":"  File \"C:\\Users\\Stass\\karman_test\\conftest.py\", line 92, in open_crm_as_admin\n    login_page.check_expected_url('https://erp-test.karman24.ru/')\n  File \"C:\\Users\\Stass\\karman_test\\pages\\base_page.py\", line 184, in check_expected_url\n    assert current_url == expected_url, \\\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":0}],"attachments":[],"parameters":[],"stepsCount":2,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"lead_page","time":{"start":1747748788371,"stop":1747748788482,"duration":111},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"contract_create_page","time":{"start":1747748788487,"stop":1747748788498,"duration":11},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"testStage":{"status":"failed","statusMessage":"AssertionError: Текущий URL не соответствует ожидаемому. Ожидалось: 'https://erp-test.karman24.ru/', Фактически: 'https://erp-test.karman24.ru/auth'","statusTrace":"login_page = <pages.login_page.admin_login.AdminLogin object at 0x000001C66F3F9190>\n\n    @pytest.fixture(scope=\"function\")\n    def open_crm_as_admin(login_page):\n        login_page.open_page()\n        with allure.step('enter correct login and password'):\n            login_page.fill_login_form_good('krivko.su@codeagency.ru', 'DLNKsfd3214$%23')\n        with allure.step('Check correct url'):\n>           login_page.check_expected_url('https://erp-test.karman24.ru/')\n\nconftest.py:92: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.login_page.admin_login.AdminLogin object at 0x000001C66F3F9190>\nexpected_url = 'https://erp-test.karman24.ru/'\n\n    def check_expected_url(self, expected_url):\n        \"\"\"\n        Проверяет, что текущий URL соответствует ожидаемому.\n        :param expected_url: Ожидаемый URL.\n        \"\"\"\n        current_url = self.driver.current_url\n>       assert current_url == expected_url, \\\n            f\"Текущий URL не соответствует ожидаемому. Ожидалось: '{expected_url}', Фактически: '{current_url}'\"\nE       AssertionError: Текущий URL не соответствует ожидаемому. Ожидалось: 'https://erp-test.karman24.ru/', Фактически: 'https://erp-test.karman24.ru/auth'\n\npages\\base_page.py:184: AssertionError","steps":[],"attachments":[{"uid":"147c342247191d1","name":"log","source":"147c342247191d1.txt","type":"text/plain","size":550}],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":1},"afterStages":[{"name":"driver::0","time":{"start":1747748768563,"stop":1747748770994,"duration":2431},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"labels":[{"name":"tag","value":"smoke('smoke test')"},{"name":"parentSuite","value":"tests.test_1_sale_in_shop"},{"name":"suite","value":"test_1_sale_in_shop"},{"name":"host","value":"LAPTOP-R624DMFA"},{"name":"thread","value":"9156-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_1_sale_in_shop.test_1_sale_in_shop"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"4eef416794ee4ba7","status":"passed","time":{"start":1747745903814,"stop":1747746012657,"duration":108843}},{"uid":"50623f9ad98a098f","status":"broken","statusDetails":"selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id: session deleted as the browser has closed the connection\nfrom disconnected: Unable to receive message from renderer\n  (Session info: chrome=136.0.7103.114)\nStacktrace:\n\tGetHandleVerifier [0x00D7FC83+61635]\n\tGetHandleVerifier [0x00D7FCC4+61700]\n\t(No symbol) [0x00BA05D3]\n\t(No symbol) [0x00B9116B]\n\t(No symbol) [0x00B90EA1]\n\t(No symbol) [0x00B8EE80]\n\t(No symbol) [0x00B8F93D]\n\t(No symbol) [0x00BB027F]\n\t(No symbol) [0x00BB2338]\n\t(No symbol) [0x00BB1140]\n\t(No symbol) [0x00BA7880]\n\t(No symbol) [0x00BA5D81]\n\t(No symbol) [0x00BA90CA]\n\t(No symbol) [0x00BA9147]\n\t(No symbol) [0x00BEA66A]\n\t(No symbol) [0x00BDE52A]\n\t(No symbol) [0x00C0D29C]\n\t(No symbol) [0x00BDE034]\n\t(No symbol) [0x00C0D514]\n\t(No symbol) [0x00C2E61B]\n\t(No symbol) [0x00C0D096]\n\t(No symbol) [0x00BDC840]\n\t(No symbol) [0x00BDD6A4]\n\tGetHandleVerifier [0x010045A3+2701795]\n\tGetHandleVerifier [0x00FFFD26+2683238]\n\tGetHandleVerifier [0x0101AA6E+2793134]\n\tGetHandleVerifier [0x00D96945+155013]\n\tGetHandleVerifier [0x00D9D02D+181357]\n\tGetHandleVerifier [0x00D874D8+92440]\n\tGetHandleVerifier [0x00D87680+92864]\n\tGetHandleVerifier [0x00D72070+5296]\n\tBaseThreadInitThunk [0x75F45D49+25]\n\tRtlInitializeExceptionChain [0x76F9D03B+107]\n\tRtlGetAppContainerNamedObjectPath [0x76F9CFC1+561]","time":{"start":1747743568634,"stop":1747743595305,"duration":26671}},{"uid":"dfae6b6fdbfd406e","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"counteragentId_0\"]\"}\n  (Session info: chrome=136.0.7103.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x0128FC83+61635]\n\tGetHandleVerifier [0x0128FCC4+61700]\n\t(No symbol) [0x010B05D3]\n\t(No symbol) [0x010F899E]\n\t(No symbol) [0x010F8D3B]\n\t(No symbol) [0x01140E12]\n\t(No symbol) [0x0111D2E4]\n\t(No symbol) [0x0113E61B]\n\t(No symbol) [0x0111D096]\n\t(No symbol) [0x010EC840]\n\t(No symbol) [0x010ED6A4]\n\tGetHandleVerifier [0x015145A3+2701795]\n\tGetHandleVerifier [0x0150FD26+2683238]\n\tGetHandleVerifier [0x0152AA6E+2793134]\n\tGetHandleVerifier [0x012A6945+155013]\n\tGetHandleVerifier [0x012AD02D+181357]\n\tGetHandleVerifier [0x012974D8+92440]\n\tGetHandleVerifier [0x01297680+92864]\n\tGetHandleVerifier [0x01282070+5296]\n\tBaseThreadInitThunk [0x767B5D49+25]\n\tRtlInitializeExceptionChain [0x777BD03B+107]\n\tRtlGetAppContainerNamedObjectPath [0x777BCFC1+561]","time":{"start":1747561681491,"stop":1747561717804,"duration":36313}},{"uid":"826da7d213ed5687","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x005CFC83+61635]\n\tGetHandleVerifier [0x005CFCC4+61700]\n\t(No symbol) [0x003F05D3]\n\t(No symbol) [0x0043899E]\n\t(No symbol) [0x00438D3B]\n\t(No symbol) [0x00480E12]\n\t(No symbol) [0x0045D2E4]\n\t(No symbol) [0x0047E61B]\n\t(No symbol) [0x0045D096]\n\t(No symbol) [0x0042C840]\n\t(No symbol) [0x0042D6A4]\n\tGetHandleVerifier [0x008545A3+2701795]\n\tGetHandleVerifier [0x0084FD26+2683238]\n\tGetHandleVerifier [0x0086AA6E+2793134]\n\tGetHandleVerifier [0x005E6945+155013]\n\tGetHandleVerifier [0x005ED02D+181357]\n\tGetHandleVerifier [0x005D74D8+92440]\n\tGetHandleVerifier [0x005D7680+92864]\n\tGetHandleVerifier [0x005C2070+5296]\n\tBaseThreadInitThunk [0x767B5D49+25]\n\tRtlInitializeExceptionChain [0x777BD03B+107]\n\tRtlGetAppContainerNamedObjectPath [0x777BCFC1+561]","time":{"start":1747553971298,"stop":1747554145314,"duration":174016}},{"uid":"7b018fe3ba96f981","status":"passed","time":{"start":1747548209798,"stop":1747548304585,"duration":94787}},{"uid":"f8574113374258fa","status":"passed","time":{"start":1747310064277,"stop":1747310162728,"duration":98451}},{"uid":"13213f2fa4e65714","status":"passed","time":{"start":1747307447526,"stop":1747307543205,"duration":95679}},{"uid":"83d060d5cb660adc","status":"passed","time":{"start":1747295200859,"stop":1747295310050,"duration":109191}},{"uid":"75263ebc44235520","status":"broken","statusDetails":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button class=\"p-button p-component p-button-icon-only p-speeddial-button p-button-rounded p-speeddial-rotate p-button-secondary p-button-outlined\" type=\"button\">...</button> is not clickable at point (3093, 234). Other element would receive the click: <div class=\"p-datatable-loading-overlay p-component-overlay\">...</div>\n  (Session info: chrome=136.0.7103.93)\nStacktrace:\n\tGetHandleVerifier [0x0128FC83+61635]\n\tGetHandleVerifier [0x0128FCC4+61700]\n\t(No symbol) [0x010B05D3]\n\t(No symbol) [0x010FECB0]\n\t(No symbol) [0x010FD054]\n\t(No symbol) [0x010FABF7]\n\t(No symbol) [0x010F9EFB]\n\t(No symbol) [0x010EE5A5]\n\t(No symbol) [0x0111D29C]\n\t(No symbol) [0x010EE034]\n\t(No symbol) [0x0111D514]\n\t(No symbol) [0x0113E61B]\n\t(No symbol) [0x0111D096]\n\t(No symbol) [0x010EC840]\n\t(No symbol) [0x010ED6A4]\n\tGetHandleVerifier [0x015145A3+2701795]\n\tGetHandleVerifier [0x0150FD26+2683238]\n\tGetHandleVerifier [0x0152AA6E+2793134]\n\tGetHandleVerifier [0x012A6945+155013]\n\tGetHandleVerifier [0x012AD02D+181357]\n\tGetHandleVerifier [0x012974D8+92440]\n\tGetHandleVerifier [0x01297680+92864]\n\tGetHandleVerifier [0x01282070+5296]\n\tBaseThreadInitThunk [0x767B5D49+25]\n\tRtlInitializeExceptionChain [0x777BD03B+107]\n\tRtlGetAppContainerNamedObjectPath [0x777BCFC1+561]","time":{"start":1747291922498,"stop":1747291960840,"duration":38342}},{"uid":"6b9dd660dd39ebbf","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'click'","time":{"start":1747145819836,"stop":1747145903404,"duration":83568}},{"uid":"a285416c67166dfc","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'click'","time":{"start":1747140783119,"stop":1747140862579,"duration":79460}},{"uid":"dbb222d25ad86f9b","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'click'","time":{"start":1747140326851,"stop":1747140406602,"duration":79751}},{"uid":"a147291b9bdcc870","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'click'","time":{"start":1747139129370,"stop":1747139209377,"duration":80007}},{"uid":"4f6be38e43bd33ea","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'click'","time":{"start":1747138001160,"stop":1747138077750,"duration":76590}},{"uid":"9344bb0be4f9d91e","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'click'","time":{"start":1747136499525,"stop":1747136577725,"duration":78200}},{"uid":"e1e8adfd30b793ff","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'click'","time":{"start":1747136216923,"stop":1747136350422,"duration":133499}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["smoke('smoke test')"]},"source":"7b87e82210905d0f.json","parameterValues":[]}